apps/evm/lib/helpers.ex

```diff
@@ -7,6 +7,9 @@ defmodule EVM.Helpers do
   use Bitwise
   alias EVM.Address
 
+  @max_uint_255 round(:math.pow(2, 255)) - 1
+  @max_256_ceiling round(:math.pow(2, 256))
```

Comment:
Just out of curiosity, does the 256 limit come from the yellow paper?

Reply:
yep, 32 bytes